### PR TITLE
Formatting/CC: Apply .clang-format to the C++ source files @open sesame 05/02 19:25

### PR DESCRIPTION
**Pull request by wooksong**
This patch applies .clang-format to all of the .cc files, which are source files written in C++.

find . -name "*.cc" -exec clang-format-15 --style=file:.clang-format {} \;

Signed-off-by: Wook Song <wook16.song@samsung.com>

### CODE REVIEW

File: daemon/model-dbus-impl.cc [hunk 1]
```diff
@@ -297,44 +284,49 @@ gdbus_cb_model_delete (MachinelearningServiceModel *obj,
...
-    .cb_data = NULL,
-    .handler_id = 0,
-  }, {
-    .signal_name = DBUS_MODEL_I_HANDLER_DELETE,
-    .cb = G_CALLBACK (gdbus_cb_model_delete),
-    .cb_data = NULL,
-    .handler_id = 0,
-  }
-};
+static struct gdbus_signal_info handler_infos[]
```
Comment (jaeyun-jung) [inline daemon/model-dbus-impl.cc:287]: can we fix this line, like pipeline-dbus-impl.cc?
```
static struct gdbus_signal_info handler_infos[] = {
  {
      {
        ....
      },
  };
```

Comment (wooksong) [reply daemon/model-dbus-impl.cc:287]: Applied.

**Comment by wooksong** [comment]
After https://github.com/nnstreamer/TAOS-CI/pull/741#pullrequestreview-1408798994 is merged, clang-format will be checked by the CI.

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**